contrib/storage-mongo/src/main/resources/bootstrap-storage-plugins.json

```diff
@@ -3,6 +3,7 @@
     "mongo" : {
       "type":"mongo",
       "connection":"mongodb://localhost:27017/",
+      "allowDiskUse": false,
```

Comment:
Is it possible that the old stored plugin objects were not able to deserialize from the ZK (based on the distributed ZK)?

Reply:
@luocooong, Thanks for the quick review.  The deserialization should only be affected if you remove config parameters. This defaults to `false`, so if the user doesn't do anything, Drill's behavior stays the same.